Makefile

```diff
@@ -473,6 +473,21 @@ clean-operator: ## Clean Operator
 	oc delete awsfederatedaccountaccess --all -n ${NAMESPACE}
 	oc delete awsfederatedrole --all -n ${NAMESPACE}
 
+.PHONY: prow-ci-predeploy
+prow-ci-predeploy: predeploy-aws-account-operator deploy-aws-account-operator-credentials create-ou-map
+	@ls deploy/*.yaml | grep -v operator.yaml | xargs -L1 oc apply -f
+
 .PHONY: prow-ci-deploy
 prow-ci-deploy: ## Triggers prow-ci build and deploy operator bash script
 	hack/scripts/prow-ci-operator-deploy.sh
+
+.PHONY: cluster-ci-entrypoint
+cluster-ci-entrypoint: ## Triggers prow-ci build and deploy operator bash script
+	hack/scripts/prow-ci-operator-deploy.sh --use-envrc --skip-cleanup --is-staging -n $(OPERATOR_NAMESPACE)
+
+.PHONY: local-ci-entrypoint
+local-ci-entrypoint: ## Triggers prow-ci build and deploy operator bash script
+	hack/scripts/prow-ci-operator-deploy.sh --use-envrc --skip-cleanup --is-local -n $(OPERATOR_NAMESPACE)
```

Comment:
consider renaming `hack/scripts/prow-ci-operator-deploy.sh` to something like  `hack/scripts/integration-test-bootstrap.sh`